docs/handling-errors.md

```diff
@@ -0,0 +1,60 @@
+![](https://i.imgur.com/zAHoOwe.png)
+<p align="center">
+    <a href="https://developers.digi.me/slack/join">
+        <img src="https://img.shields.io/badge/chat-slack-blueviolet.svg" alt="Developer Chat">
+    </a>
+    <a href="LICENSE">
+        <img src="https://img.shields.io/badge/license-apache 2.0-blue.svg" alt="MIT License">
+    </a>
+    <a href="#">
+    	<img src="https://img.shields.io/badge/build-passing-brightgreen.svg">
+    </a>
+    <a href="https://www.typescriptlang.org/">
+        <img src="https://img.shields.io/badge/language-typescript-ff69b4.svg" alt="Typescript">
+    </a>
+    <a href="https://developers.digi.me/">
+        <img src="https://img.shields.io/badge/web-digi.me-red.svg" alt="Web">
+    </a>
+</p>
+
+<br>
+
+# Handling Errors
+
+By default, the SDK retires to make calls twice before giving up. This is something that can be configured when the SDK is [initialised](./initialise-sdk.md).
+
+## Error Types
+
+These errors with be thrown if there is an error during SDK execution.
+
+`HTTPError` - This is the default type of error we throw when we receive a server side error.
+
+`ParameterValidationError` - Thrown if the parameters that's been passed in to a function is invalid.
```

Comment:
`parameters that have been passed`, `parameters that's been passed` doesn't read right.